python_modules/dagster/dagster/core/definitions/partition.py

```diff
@@ -238,6 +247,32 @@ def get_partitions(
     ) -> Sequence[Partition[str]]:
         return self._partitions
 
+    def get_partition_keys_in_range(self, start: str, end: str) -> Sequence[str]:
```

Comment:
not blocking, and very much a silly quibble, but I'm not sure if I would expect the partitions defined by a static partitions definition to have a defined order. They implicitly do, because we take in a List[str] not Set[str], I just felt vaguely weird seeing this, as I would have intuitively expected StaticPartitionsDefinition(["car", "truck"]) == StaticPartitionsDefinition(["truck", "car"]).

People that want that to be the case don't have to use this function, so it's probably fine, just thought I'd flag.

Reply:
Hmm yeah - thoughts on whether there are any `StaticPartitionsDefinitions` that make sense to be ordered?

Reply:
I'll just put this on TimeWindowPartitionsDefinition for now